msvc/sdk-build.props

```diff
@@ -24,6 +24,10 @@
     </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Condition="'$(BuildingFrameworksCore)' != 'true'" Include="WinObjC.Frameworks.UWP" Version="*" />
```

Comment:
This is an interesting approach!

Do you think it should go down with the other PackageReferences?